_posts/2016/03/2016-03-08-big-data-in-biology-summer-school.md

```diff
@@ -4,7 +4,7 @@ authors: ["Rayna Harris"]
 title: "3rd Annual Big Data in Biology Summer School"
 date: 2016-03-08
 time: "13:30:00"
-category: ["UTexas"]
+category: ["UNIVERSITY OF TEXAS - AUSTIN"]
```

Comment:
The [other post has][1] `University of Texas - Austin`, so I think we want that here (and not your UPPERCASE form).

[1]: https://github.com/swcarpentry/website/blob/fc3e71b191ca141e0d6b21bf452dcbc6dc83f08f/_posts/2012/12/2012-12-10-things-are-going-well-in-texas-on-ada-lovelaces-birthday.html#L7